### PR TITLE
Release notes for BRP

### LINKED ISSUE #1696: Write release notes for PR #14880: Initial implementation of the Bevy Remote Protocol (Adopted)
https://github.com/bevyengine/bevy/pull/14880 needs release notes for the upcoming Bevy release!

Please reply below if you'd like to write these notes. 
While the author(s) of the PR often have the context, knowledge and motivation to draft the release notes for their feature, anyone can contribute release notes!

------

Release notes should:

1. Clearly motivate the change.
2. Be written in a way that is understandable by the average Bevy user: some programming background and a general understanding of games.
3. Show off the coolest features of the PR. Screenshots are awesome, but elegant APIs are also welcome!
4. If this was a perf-centric PR, quantify the performance improvements. Graphs and statistics work well for this.

We can help you revise the release notes: a rough draft alone is incredibly useful :)
Your expertise is invaluable for contextualizing the changes; we'll work with you to bring the technical writing up to par.

To submit your release notes, modify `/home/alice/Documents/bevy-website/release-content/0.15/release-notes/14880_Initial_implementation_of_the_Bevy_Remote_Protocol_Adopted.md` and submit a PR.
In that PR, please mention this issue with the `Fixes #ISSUE_NUMBER` keyphrase so it gets closed automatically.

### PR DESCRIPTION
**Pull request by mweatherley**
This is pretty basic; I didn't want to go into too much detail, since I think we should generally point people toward the module documentation for that. I think a little bit more motivation in terms of what this can be used for would probably be welcome, but I struggled to spark my imagination, so any help in that area would be appreciated :)

Fixes #1696.

### CODE REVIEW

**Comment by alice-i-cecile** [comment]
Yeah, motivation is the primary focus of this particular release note IMO :)

File: release-content/0.15/release-notes/14880_Initial_implementation_of_the_Bevy_Remote_Protocol_Adopted.md [hunk 1]
```diff
@@ -1,4 +1,65 @@
-<!-- Initial implementation of the Bevy Remote Protocol (Adopted) -->
-<!-- https://github.com/bevyengine/bevy/pull/14880 -->
+We have introduced the Bevy Remote Protocol, which allows the ECS of a running
```
Comment (alice-i-cecile) [inline release-content/0.15/release-notes/14880_Initial_implementation_of_the_Bevy_Remote_Protocol_Adopted.md:1]:
```suggestion
The Bevy Remote Protocol allows the ECS of a running
```

File: release-content/0.15/release-notes/14880_Initial_implementation_of_the_Bevy_Remote_Protocol_Adopted.md [hunk 2]
```diff
@@ -1,4 +1,65 @@
...
+Bevy application to be interacted with remotely. This can be used, for example,
+to inspect and edit entities and their components at runtime. We anticipate 
+that this will be used primarily to create things like inspectors for editing
```
Comment (alice-i-cecile) [inline release-content/0.15/release-notes/14880_Initial_implementation_of_the_Bevy_Remote_Protocol_Adopted.md:4]: I think that calling out the multi-process editor architecture is helpful here. Lots of hedging on "experimental" please.

File: release-content/0.15/release-notes/14880_Initial_implementation_of_the_Bevy_Remote_Protocol_Adopted.md [hunk 3]
```diff
@@ -1,4 +1,65 @@
...
+and debugging.
 
-<!-- TODO -->
+The default methods included with the associated plugin include:
```
Comment (alice-i-cecile) [inline release-content/0.15/release-notes/14880_Initial_implementation_of_the_Bevy_Remote_Protocol_Adopted.md:7]:
```suggestion
For now, you can use BRP to:
```
Remember to change the tense of the follow-up items to match.

File: release-content/0.15/release-notes/14880_Initial_implementation_of_the_Bevy_Remote_Protocol_Adopted.md [hunk 4]
```diff
@@ -1,4 +1,65 @@
...
+  their associated values;
+- creating a new entity with a given set of component values;
+- despawning an entity;
+- inserting a set of components into an entity;
+- removing a set of components from an entity;
+- reparenting one or more entities;
+- listing the components registered in the ECS or present on an entity.
+
+Details on these methods are available in the `bevy_remote` module documentation.
+Additionally, custom methods can also be written.
```
Comment (alice-i-cecile) [inline release-content/0.15/release-notes/14880_Initial_implementation_of_the_Bevy_Remote_Protocol_Adopted.md:19]: I don't think this is helpful without more explanation. Either explain, link to docs or cut.

**Comment by mnmaita** [comment]
Should we add some note around this change? #16198

Something along the lines of "these systems will run after all user systems" (but proper 😅).

**Comment by mweatherley** [comment]
> Should we add some note around this change? #16198
> 
> Something along the lines of "these systems will run after all user systems" (but proper 😅). Might be irrelevant though.

My inclination is to think this is too particular for the release notes; I didn't say anything about the streaming methods either, for instance.

**Review by alice-i-cecile — APPROVED** [review]

**Pull request merged**